avoidance/src/common.cpp

```diff
@@ -226,6 +226,11 @@ float wrapAngleToPlusMinusPI(float angle) { return angle - 2.0f * M_PI_F * std::
 
 float wrapAngleToPlusMinus180(float angle) { return angle - 360.f * std::floor(angle / 360.f + 0.5f); }
 
+float angleDifference(float a, float b) {
+  float angle = a - b;
+  return std::abs(angle >= 0.f ? (angle < 180.f) ? angle : angle - 360.f : (angle >= -180.f) ? angle : angle + 360.f);
```

Comment:
Maybe we don't need to do an `abs` here, since you're squaring it down below?

(and then the doc string can change too)

Reply:
done